docs/API/InstrumentFormat.md

```diff
@@ -109,6 +143,15 @@ the following format:
 `Description`: The human readable description of this element, such
                as question or label text. This may or may not be required
                depending on element type.
+               
+`DisplayIf`: A formula that returns true when the element should be
+             displayed. Primarily used to display fields based on
+             candidate context or previous-answers. Defaults to `true`.
+             
+`HiddenSurvey`: A formula that returns true if the element should be hidden
+                from respondants of a survey. Defaults to `false`.
```

Comment:
I don't think "Survey" is defined anywhere in this document, so this needs some clarification as to what it means.

Reply:
good point